scripts/globals/weaponskills/requiescat.lua

```diff
@@ -15,39 +15,21 @@ require("scripts/globals/weaponskills")
 local weaponskill_object = {}
 
 weaponskill_object.onUseWeaponSkill = function(player, target, wsID, tp, primary, action, taChar)
-
     local params = {}
     params.numHits = 5
-
-    params.ftp100 = 1
-    params.ftp200 = 1
-    params.ftp300 = 1
-
-    params.str_wsc = 0.0
-    params.dex_wsc = 0.0
-    params.vit_wsc = 0.0
-    params.agi_wsc = 0.0
-    params.int_wsc = 0.0
-    params.mnd_wsc = 0.0 + (player:getMerit(xi.merit.REQUIESCAT) * 0.17)
+    params.ftp100 = 1.0 params.ftp200 = 1.0 params.ftp300 = 1.0
+    params.str_wsc = 0.0 params.dex_wsc = 0.0 params.vit_wsc = 0.0
+    params.agi_wsc = 0.0 params.int_wsc = 0.0 params.mnd_wsc = player:getMerit(xi.merit.REQUIESCAT) * 0.17
     params.chr_wsc = 0.0
-
+    params.crit100 = 0.0 params.crit200 = 0.0 params.crit300 = 0.0
     params.canCrit = false
-    params.crit100 = 0.0
-    params.crit200 = 0.0
-    params.crit300 = 0.0
-
-    params.acc100 = 0.0
-    params.acc200 = 0.0
-    params.acc300 = 0.0
-
-    params.atk100 = 0.8
-    params.atk200 = 0.9
-    params.atk300 = 1.0
-
+    params.acc100 = 0.0 params.acc200 = 0.0 params.acc300 = 0.0
+    params.atk100 = 0.8 params.atk200 = 0.9 params.atk300 = 1.0
     params.formless = true
```

Comment:
Can you add a TODO to verify this "formless" check, (planning ahead for my future refactor) 

thanks.

Reply:
Thanks!